File: 2149-remove-colored-pieces-if-both-neighbors-are-the-same-color/remove-colored-pieces-if-both-neighbors-are-the-same-color.c
bool winnerOfGame(char *colors) {
    int a = 0;
    int b = 0;
    char* cur = colors;

   while (*(cur) != '\0' && *(cur + 1) != '\0' && *(cur + 2) != '\0')
 {
        if (*(cur) == 'A' && *(cur + 1) == 'A' && *(cur + 2) == 'A') {
            a += 1;
        } else if (*(cur) == 'B' && *(cur + 1) == 'B' && *(cur + 2) == 'B') {
            b += 1;
        }
        cur++;
    }

    return a > b;
}

File: 2149-remove-colored-pieces-if-both-neighbors-are-the-same-color/remove-colored-pieces-if-both-neighbors-are-the-same-color.py
class Solution:
    def winnerOfGame(self, colors: str) -> bool:
        a, b= 0, 0
        for i in range(len(colors)-2):
            if colors[i:i+3]=="AAA":
                a+=1
            elif colors[i:i+3]=='BBB':
                b+=1
        if a==b:
            return False
        elif a>b:
            return True
        else:
            return False
